mmc-github/src/main/java/info/novatec/metricscollector/github/GithubServiceScheduler.java

```diff
@@ -0,0 +1,52 @@
+package info.novatec.metricscollector.github;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
+@ConfigurationProperties(prefix = "github.project")
+public class GithubServiceScheduler {
+
+    @Autowired
+    private GithubCollector githubCollector;
+
+    @Autowired
+    private GithubService githubService;
+
+    private static final String GITHUB_MAIN_URL = "https://github.com/";
+
+    private List<String> urls = new ArrayList<>();
+
+    /**
+     * Accesses the urls from application.yml
+     *
+     * @return List of Strings with the respective urls
+     */
+    public List<String> getUrls() {
+        return this.urls;
```

Comment:
Is it invoked external? If not, the method can be removed. access "urls" directly. 
Twi things, if it is used by another class..
1. data encapsulation can be set to package private. There should never be a class outside the github-package, which uses github-collector/service
2. "this." is not necessary if there is no equal parameter